src/renderer/src/helpers/StringHelper.ts

```diff
@@ -0,0 +1,15 @@
+export class StringHelper {
+  static formatPrice(price) {
+    if (!price) {
+      return '$0.00'
+    }
+    return price.toLocaleString('en-US', { style: 'currency', currency: 'USD' })
+  }
```

Comment:
Do you need this function? Could it be that the `FilterHelper.currency` function doesn't do the same thing?